src/core/actor.h

```diff
@@ -28,7 +28,7 @@ class PiVal {
   PiVal(int player, float value, std::vector<float> pi) {
     reset();
     playerId = player;
-    value = value;
+    this->value = value;
```

Comment:
this should match the lines above and below

Reply:
Which lines?